common/libs/transaction.js

```diff
@@ -137,7 +137,7 @@ export async function generateCompleteTransactionFromRawTransaction(
     nonce: cleanHex(nonce),
     gasPrice: cleanHex(gasPrice.toString(16)),
     gasLimit: cleanHex(gasLimit.toString(16)),
-    to: cleanHex(to),
+    to: toChecksumAddress(cleanHex(to)),
```

Comment:
I couldn't find _where_ in code this happens, but all of the transactions I generated showed the checksum'd version in v3, whereas v4 was allowing all lower / all caps versions into the raw tx.